packages/features/shell/Shell.tsx

```diff
@@ -506,60 +482,59 @@ function UserDropdown({ small }: UserDropdownProps) {
                 <DropdownMenuItem>
                   <DropdownItem
                     type="button"
-                    StartIcon={(props: { className?: string }) => (
-                      <UserIcon className={classNames("text-default", props.className)} aria-hidden="true" />
-                    )}
+                    CustomStartIcon={
+                      <Icon name="user" className={classNames("text-default")} aria-hidden="true" />
+                    }
                     href="/settings/my-account/profile">
                     {t("my_profile")}
                   </DropdownItem>
                 </DropdownMenuItem>
                 <DropdownMenuItem>
                   <DropdownItem
                     type="button"
-                    StartIcon={(props: { className?: string }) => (
-                      <Settings className={classNames("text-default", props.className)} aria-hidden="true" />
-                    )}
+                    CustomStartIcon={
+                      <Icon name="settings" className={classNames("text-default")} aria-hidden="true" />
+                    }
                     href="/settings/my-account/general">
                     {t("my_settings")}
                   </DropdownItem>
                 </DropdownMenuItem>
                 <DropdownMenuItem>
                   <DropdownItem
                     type="button"
-                    StartIcon={(props: { className?: string }) => (
-                      <Moon className={classNames("text-default", props.className)} aria-hidden="true" />
-                    )}
+                    CustomStartIcon={
+                      <Icon name="moon" className={classNames("text-default")} aria-hidden="true" />
+                    }
                     href="/settings/my-account/out-of-office">
                     {t("out_of_office")}
                   </DropdownItem>
                 </DropdownMenuItem>
                 <DropdownMenuSeparator />
                 <DropdownMenuItem>
                   <DropdownItem
-                    StartIcon={() => <Discord className="text-default h-4 w-4" />}
+                    CustomStartIcon={<Discord />}
```

Comment:
Found this issue. Fixing..
<img width="246" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/356ef441-5cd3-4fff-a36d-7ce22de7149f">